File: fhg-api/src/main/java/de/jbamberger/api/ParseException.java
package de.jbamberger.api;

import java.io.IOException;

/**
 * @author Jannik Bamberger (dev.jbamberger@gmail.com)
 */

public class ParseException extends IOException {
    public ParseException() {
        super();
    }

    public ParseException(String message) {
        super(message);
    }

    public ParseException(String message, Throwable cause) {
        super(message, cause);
    }

    public ParseException(Throwable cause) {
        super(cause);
    }
}

File: fhg-api/src/main/java/de/jbamberger/api/ParseException.kt
package de.jbamberger.api

import java.io.IOException

/**
 * @author Jannik Bamberger (dev.jbamberger@gmail.com)
 */

class ParseException : IOException {
    constructor() : super() {}

    constructor(message: String) : super(message) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}

    constructor(cause: Throwable) : super(cause) {}
}
